File: pyhton64/nivel2/exer16.py
valor1 = int(input('Primeiro valor: '))
valor2 = int(input('Segundo valor: '))

if valor1 > valor2:
    print(f'Maior {valor1}\nMenor {valor2}')
elif valor2 > valor1:
    print(f'Maior {valor2}\nMenor {valor1}')
else:
    print('São iguais')

File: java64/nivel2/exer16.java
import javax.swing.JOptionPane;

public class exer16 {
    public static void main(String[] args) {
        int valor1, valor2;

        valor1 = Integer.parseInt(JOptionPane.showInputDialog(null, "Valor 1"));
        valor2 = Integer.parseInt(JOptionPane.showInputDialog(null, "Valor 2"));

        if (valor1 > valor2) JOptionPane.showMessageDialog(null, "Valor 1 é maior que Valor 2");
        if (valor1 < valor2) JOptionPane.showMessageDialog(null, "Valor 2 é maior que Valor 1");
        if (valor1 == valor2) JOptionPane.showMessageDialog(null, "Os dois valores são iguais");
    }
}